<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>footsall_arrange</class>
 <widget class="QWidget" name="footsall_arrange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="our_team_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sakkal Majalla</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>بازی های ما</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tournoments">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tour_name">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>نام تورنومنت را انتخاب نمایید.</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="our_games">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>691</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string> برای تغییر چیزی تغییرات را در جدول اعمال کنید .</string>
   </property>
   <column>
    <property name="text">
     <string>تیم اول</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>امتیاز تیم 1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>امتیاز تیم 2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تیم دوم</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>تاریخ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ساعت</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>حذف</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>اعمال تغییرات</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QComboBox" name="team2"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="team1"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>تیم اول</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>تیم دوم</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="toolTip">
       <string extracomment="لطفا تاریخ و ساعت مسابقه ی خود را به میلادی وارد نمایید"/>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="minimumTime">
       <time>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::AmPmSection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>زمان مسابقه</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="addgame">
      <property name="text">
       <string>اضافه کردن بازی</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
